azure-cs-aks-helm/MyCluster.cs

```diff
@@ -0,0 +1,89 @@
+// Copyright 2016-2021, Pulumi Corporation.  All rights reserved.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+using Pulumi.AzureAD;
+using Pulumi.AzureNative.ContainerService;
+using Pulumi.AzureNative.ContainerService.Inputs;
+using Pulumi.AzureNative.Resources;
+
+class MyCluster {
+    public Pulumi.Output<string> ClusterName { get; set; }
+    public Pulumi.Output<string> Kubeconfig { get; set; }
+
+    public Pulumi.Kubernetes.Provider Provider { get; set; }
+
+    public MyCluster(MyConfig cfg) {
+        var resourceGroup = new ResourceGroup("rg");
+
+        var adApp = new Pulumi.AzureAD.Application("app");
+
+        var adSp = new ServicePrincipal("service-principal", new ServicePrincipalArgs() {
```

Comment:
Let's stick to our usual C# formatting:

```suggestion
        var adSp = new ServicePrincipal("service-principal", new ServicePrincipalArgs
        {
```